xayn-ai/src/coi/point.rs

```diff
@@ -71,17 +72,21 @@ impl From<PositiveCoi_v0_2_0> for PositiveCoi {
     }
 }
 
-#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[derive(Clone, Debug, Derivative, Deserialize, Serialize)]
+#[derivative(PartialEq)]
 pub(crate) struct NegativeCoi {
     pub(super) id: CoiId,
     pub(super) point: Embedding,
+    #[derivative(PartialEq = "ignore")]
+    pub(super) last_view: SystemTime,
```

Comment:
Ignoring fields for equality comparison is generally a bad idea (it e.g. can lead so surprising results when, e.g. de-duplicating a list). But I guess it might be fine here?

Reply:
the statistics of a coi can be viewed as meta data about the coi which doesn't change what constitutes the coi itself. in that sense it shouldn't affect the comparison, this is also how we handle other stats in the cois.

Reply:
Thanks for the answer, I wanted to ask you about it today